### ISSUE TITLE
Disabling 'x-managed-by' header

### ISSUE THREAD

**Issue by m0uneer** [Romakita/ts-express-decorators#99]
Hi Romain :),

Is there a way to disable `x-managed-by` header?
Express provides a flag to handle `x.powered-by` header
```javascript
    app.set('x-powered-by', false);
```
So, I suggest the same way to handle `x-managed-by` header
```javascript
    this.set('x-managed-by', false);
```

Thanks for your help

**Comment by Romakita** [comment]
Hi @m0uneer ,
I prefer just to remove this header ^^.

**Comment by Romakita** [comment]
Fix with v2.0.15
